explorer/explorerroutes.go

```diff
@@ -299,6 +289,6 @@ func (exp *explorerUI) errorPage(w http.ResponseWriter, code string, message str
 		str = "Something went very wrong if you can see this, try refreshing"
 	}
 	w.Header().Set("Content-Type", "text/html")
-	w.WriteHeader(http.StatusNotFound)
```

Comment:
Oh, I meant 500 only when the template execution fails.  It can be 404 when the user searches for a non-existent item.